cupy/core/core.pyx

```diff
@@ -1933,89 +1942,23 @@ cpdef ndarray tensordot_core(
         c.shape = (n, m)
 
     # Be careful that cuBLAS uses the FORTRAN-order matrix representation.
-    if k == 1:
```

Comment:
It is historical reasons.
We had believed that `semm` can not calculate between a matrix and a vector.